src/api/server.ts

```
@@ -1,6 +1,11 @@
 import express from "express";
 import cors from "cors";
 import { getLeaderboard } from "../db/student_stats.model.js";
+import {
+  leaderboardSecurityMiddlewares,
+  apiSecurityMiddlewares,
+  securityLogger
```

ESLint

[error] Insert `,`

(prettier/prettier)